### ISSUE TITLE
Clearly indicate when a post is coming from someone's RSS feed

### ISSUE THREAD

**Issue by FerociousTruth** [Discordius/Lesswrong2#181]
I find it confusing that there is not visible distinction between posts that are written directly on LessWrong and ones that are mirrored from someone's RSS feed.  For example, [Schelling fences on slippery slopes](https://www.lesserwrong.com/posts/Kbm6QnJv9dgWsPHQP/schelling-fences-on-slippery-slopes) and [Against Individual IQ Worries](https://www.lesserwrong.com/posts/AmaWMMWPzuQ62Ernf/against-individual-iq-worries) both appear to be posted directly on LessWrong even though one comes from Scott's blog and the other is, I'm pretty sure, an original post from the old LessWrong site.

I'd like to see a clear indication that a post is coming from a feed at the top of the post, similar to how when people cross-post they will often indicate this at the top.  It should include a link the the original site.  Additionally, I *may* want an indication of this on pages that list posts so that I can easily pick out content that I wouldn't have already read elsewhere.

Clearly indicating that posts are coming from another site would be a prerequisite for me to use the RSS feed feature for my own blog.

**Comment by Discordius** [comment]
Yep, agree with this. The design for this is fairly straightforward I think. I just haven't gotten around to it.
